bin/git-summary

```diff
@@ -136,10 +143,15 @@ format_authors() {
   # a rare unicode character is used as separator to avoid conflicting with
   # author name. However, Linux column utility will escape tab if separator
   # specified, so we do unesaping after it.
-  LC_ALL=C awk '
+  echo "${1:?}" | LC_ALL=C awk '
+  BEGIN {
+     commitLimit = strtonum('"${AUTHOR_COMMIT_LIMIT}"');
+  }
   { args[NR] = $0; sum += $0 }
   END {
     for (i = 1; i <= NR; ++i) {
+      if (strtonum(args[i]) < commitLimit)
```

Comment:
The `strtonum` seems to be GNU awk extension, can we find an alternative?

Reply:
Seems like it's unnecessary to call `strtonum` at all, the comparison should work anyway.